update package for cisco asa

v-laanjana opened the pull request:
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution cisco asa
See guidance below
Version Updated:
2.0.4
See guidance below
Testing Completed:
The link for test results is
[Sentinel Solution Migratio](https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B0A77223C-9291-43A3-BBA6-60A0F1ADC775%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-cisco%20asa.docx&action=default&mobileredirect=true&share=IQE8IncKkZKjQ7umYKDxrcd1ASkVtycDJtMoZZ4-nr8K65g)n


See guidance below
This is an isv solution.

v-laanjana committed:
update package for cisco asa

v-laanjana committed:
Merge branch 'master' into v-laanjana/solution-update-cisco-asa-with-pb

v-laanjana committed:
update solution migration  cisco asa !

v-atulyadav reviewed (commented):
Hi @v-laanjana , please add `status: Available` in all analytical rules

v-laanjana committed:
update status: Available

v-atulyadav reviewed (commented):
Hi @v-laanjana, please remove `-preview` from SolutionId

![image](https://user-images.githubusercontent.com/104008048/185383174-d4f1650e-3e07-44cd-9d26-e6acec59c65a.png)

v-laanjana committed:
updated solution

v-amolpatil committed:
added skip validation change for analytical rules

v-amolpatil committed:
Merge branch 'master' into v-laanjana/solution-update-cisco-asa-with-pb

v-amolpatil committed:
Merge branch 'master' into v-laanjana/solution-update-cisco-asa-with-pb

v-amolpatil committed:
reverted change in skip files

removed id from detectionTemplateSchemaValidation file

Minor change in id

removed some tag

reverted file change

v-amolpatil committed:
Update SkipIdValidationsTemplates.json

This reverts commit 703fd8d8d2dceac75164cbfe7ee9805c2fa49ee7.

v-amolpatil committed:
reverted all files changes for skip validation

v-amolpatil committed:
Merge branch 'master' into v-laanjana/solution-update-cisco-asa-with-pb
